specification/cognitiveservices/data-plane/AnomalyDetector/preview/v2.0-preview/RootCauseAnalysis.json

```diff
@@ -0,0 +1,2143 @@
+        [
+          "CN",
+          "__SUM__",
+          0.3,
+          302,
+          1000
+        ],
+        [
+          "CN",
+          "ios",
+          0.3,
+          302,
+          1000
+        ],
+        [
+          "CN",
+          "Android",
+          0,
+          0,
+          1000
+        ],
+        [
+          "US",
+          "__SUM__",
+          0.4,
+          401,
+          1000
+        ],
+        [
+          "JP",
+          "__SUM__",
+          0.009,
+          9,
+          1000
+        ],
+        [
+          "DE",
+          "__SUM__",
+          0.011,
+          11,
+          1000
+        ],
+        [
+          "CA",
+          "__SUM__",
+          0.01,
+          1,
+          1000
+        ],
+        [
+          "__SUM__",
+          "__SUM__",
+          1,
+          1000,
+          1000
+        ]
+      ]
+    },
+    {
+      "partitionIdentity": "PartitionIdentity5",
+      "timestamp": "2021-11-27T00:00:00Z",
+      "data": [
+        [
+          "CN",
+          "__SUM__",
+          0.299,
+          299,
+          1000
+        ],
+        [
+          "CN",
+          "ios",
+          0.31,
+          310,
+          1000
+        ],
+        [
+          "CN",
+          "Android",
+          0.001,
+          1,
+          1000
+        ],
+        [
+          "US",
+          "__SUM__",
+          0.401,
+          401,
+          1000
+        ],
+        [
+          "JP",
+          "__SUM__",
+          0.01,
+          10,
+          1000
+        ],
+        [
+          "DE",
+          "__SUM__",
+          0.01,
+          10,
+          1000
+        ],
+        [
+          "CA",
+          "__SUM__",
+          0.01,
+          1,
+          100
+        ],
+        [
+          "__SUM__",
+          "__SUM__",
+          1,
+          1000,
+          1000
+        ]
+      ]
+    }
+  ],
+  "kind": "multidimensional"
+}
+    },
+    "MultiDimRCAUploadDataRequest": {
+      "required": [
+        "value"
+      ],
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/RootCauseAnalysisUploadDataRequest"
+        }
+      ],
+      "properties": {
+        "value": {
+          "description": "Multidimensional root cause analysis data",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/MultiDimRCADataPartition"
+          }
+        }
+      },
+      "x-ms-discriminator-value": "multidimensional"
+    },
+    "RootCauseAnalysisDataMeta": {
+      "required": [
+        "kind"
+      ],
+      "type": "object",
+      "properties": {
+        "kind": {
+          "description": "Root cause analysis kind",
+          "enum": [
+            "multidimensional"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "RootCauseAnalysisType",
+            "modelAsString": true
+          }
+        }
+      },
+      "discriminator": "kind"
+    },
```

Comment:
Here is another schema with a discriminator with only one "kind". I really want to understand why/if this is needed.

Reply:
kind is a field to describe what kind of RCA this instance is . Currently, the kind is multidimensional root cause analysis. In the future, we plan to provide cross metric root cause analysis APIs